.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
-  build:
+  ci:
```

Comment:
Requires a change in the protected branches

Reply:
Done